Userland/Libraries/LibJS/Runtime/Temporal/CalendarPrototype.cpp

```diff
@@ -120,8 +121,7 @@ JS_DEFINE_NATIVE_FUNCTION(CalendarPrototype::year)
 
     auto temporal_date_like = vm.argument(0);
     // 4. If Type(temporalDateLike) is not Object or temporalDateLike does not have an [[InitializedTemporalDate]] or [[InitializedTemporalYearMonth]] internal slot, then
-    // TODO PlainYearMonth objects
-    if (!temporal_date_like.is_object() || !is<PlainDate>(temporal_date_like.as_object())) {
+    if (!temporal_date_like.is_object() || !(is<PlainDate>(temporal_date_like.as_object()) || is<PlainYearMonth>(temporal_date_like.as_object()))) {
```

Comment:
Same in the `LibJS: Handle ZonedDateTime in the Calendar.prototype getters` commit as above